src/core/Akka.TestKit/TestKitBase_Expect.cs

```diff
@@ -590,15 +581,14 @@ public ValueTask<T> ExpectMsgAnyOfAsync<T>(IEnumerable<T> messages, Cancellation
         {
             return ExpectMsgAllOfAsync(messages, cancellationToken)
                 .ToListAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.